helpers/validation.js

```diff
@@ -9,3 +9,17 @@ export const isObjectId = (value, helpers) => {
   // Return the value unchanged
   return value;
 };
+
+export const idValidationSchema = Joi.string().custom(isObjectId, 'Invalid id');
+
+export const phoneNumberPattern = /^\+38\(0[3-9]\d\)\d{7}$/;
```

Comment:
This regular expression is only for Ukraine phone numbers, but we can add other countries for addresses. So, maybe, we can allow other country's phone numbers?

Reply:
I guess for the first version of our web app it's enough to choose one country, one currency and phone number types. The region can be expanded in the future